### PR TITLE
feat: update "who's using" section with Pinata video and link to ecosystem directory

### PR DESCRIPTION
**Pull request by jessicaschilling**
This PR replaces the old "Who's already using IPFS?" link to the logo parade with two items:
1. Screenshot of Pinata "Builders Series" video, to open in a modal displaying similarly to videos on the IPFS blog 
2. Link and teaser text to the interactive ecosystem directory at ecosystem.ipfs.io

Also updates press kit page's link to the old raster diagram to instead go to the ecosystem directory.

image

**Remaining to do:**

- [x] Make the video display in the same modal window as videos on the IPFS blog (minus the date, tags, and sharing items at the bottom of the modal)
- [x] Make the "play button" overlay display correctly - e.g. in the same way as in the "Latest videos" section below this section
- [x] Finalize video YouTube URL (right now it's linked to a different video, Pinata video isn't live yet)

Closes https://github.com/ipfs/ipfs-website/issues/60. Closes https://github.com/ipfs/ipfs-website/issues/61.

### CODE REVIEW

**Comment by jessicaschilling** [comment]
@zebateira, is there any chance you'd be available to help with the first two to-do items in the next few days, so I don't need to disrupt your flow when we're approved to add these items? Thank you!

**Comment by zebateira** [comment]
@jessicaschilling sounds good 👍 I can work on it. Do we also want to add tracking to the CTA?

**Comment by jessicaschilling** [comment]
@zebateira I did add two uniquely-named instances of `onCTAClick` — is there anything else that needs to be done on the Countly side?

**Comment by zebateira** [comment]
@jessicaschilling oh that's it then! sorry, missed it the first time around 👍

**Comment by zebateira** [comment]
@jessicaschilling just pushed the required changes. Let me know what you think!

File: components/VideoModal.vue [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+          rounded
+          w-11/12
+          md:w-4/5
+          max-w-screen-lg max-h-screen
+          overflow-y-auto
+          p-4
+          lg:p-8
+        "
+      >
+        <h1 class="type-h5 text-xl text-primary mr-4">
```
Comment (jessicaschilling) [inline components/VideoModal.vue:24]:
```suggestion
        <h1 class="type-h5 font-semibold text-xl text-primary mr-4">
```

**Comment by jessicaschilling** [comment]
@zebateira This is great - thanks so much! We'll be all ready to go once these items get approved and then I'll circulate this for approval. No need for you to pay any more attention going forward 😊

**Comment by jessicaschilling** [comment]
@atopal @momack2 — Requesting your review for updating ipfs.io with links to Pinata video and ecosystem directory. 

Suggest we deploy this website change circa 15 July to be in line with formal video launch, and to continue along the soft-launch pathway for the ecosystem directory.

Thank you!

**Review by atopal — APPROVED** [review]
Yes, let's ship it! 🎉

**Comment by jessicaschilling** [comment]
Merging per OOB team check-in.

**Comment by momack2** [comment]
Nice! @Jessica Schilling ***@***.***> - can you track the number
of interactions with this video and the button to the ecosystem directory
and share with the team in a week or two? Hopeful it's a great success
story for adding this explanatory content to the website!

**Comment by jessicaschilling** [comment]
@momack2 Certainly, but please note that this would be a measurement of organic discovery _only_, since we haven't yet formally promoted the ecosystem directory in a hard launch. In the case of the video, overall views in YouTube's dashboard is probably a more useful metric, since that includes views from all sources.

@zebateira, can you please confirm that `openVideoModal` is registering as an event? I'm not seeing it yet, but understood we may be dealing with a lag in reporting data.

**Comment by zebateira** [comment]
> @zebateira, can you please confirm that openVideoModal is registering as an event? I'm not seeing it yet, but understood we may be dealing with a lag in reporting data.

We're using the CTA event for that with `ui=who/meet-builders`.
I created a widget for this in the dashboard and the numbers are coming up:

image

**Comment by jessicaschilling** [comment]
@zebateira Thanks for setting up the widget ... you beat me to it! 😊 

@momack2 - This will be a section in the weekly email report going forward. (Countly's visual rendering of those weekly reports isn't always fantastic, so if the next report is tough to read/parse, I'll circulate numbers separately.) This'll give us some good baseline metrics as well as enable us to see (in the case of the video) what percentage of overall YouTube traffic is coming from the website. Ecosystem directory internal metrics should be online in the next few days (that's one of our hard requirements for a formal launch), so we can do some similar comparisons for that as well. Finally, once we've got a solid stretch of time under our belt with these new metrics, we could start thinking of some A/B testing — maybe seeing the impact of moving this section higher up on the page.

Thanks both!

**Pull request merged**